desktop/cmp/input/Select.js

```diff
@@ -257,8 +264,7 @@ export class Select extends HoistInput {
         if (props.hideDropdownIndicator) {
             rsProps.components = {
```

Comment:
Since we are redoing this code to always install `components`  think it would be much simpler to just consult this prop on line 226 with a ternary.

Reply:
(and of course we need to spread in ...rsProps.components  on line 226 as well -- that would have been a regression here when the dropdownIndicator was not hidden)